.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,27 @@
+---
+name: Sphinx build
+
+on: push
```

Comment:
Maybe for now simplest approach is to have a manual worflow-dispatch, rather than on push or anything sophisticated.  Then we can manually run it when we tag a new version.  It also makes it easy to fix docs and rebuild without making a new tag.

Reply:
I changed this to
```yml
on: workflow_dispatch
```
now